version/version.go

```diff
@@ -199,6 +397,9 @@ func parseInto(result *Version, input string) error {
 	}
 
 	colon := strings.Index(trimmed, ":")
+	if (colon >= 9) {
+		return fmt.Errorf("epoch too big or not an integer")
```

Comment:
The epoch can be any unsigned integer; a valid version can be `10000000000:1.0-1`

Reply:
Such epoch is valid, but not supported. It does not fit into 32bit integer, so most parsers would have problems (comparing it) anyways.